metadata.json

```diff
@@ -19,6 +19,18 @@
         "6",
         "7"
       ]
+    },
+    { "operatingsystem": "Debian",
+      "operatingsystemrelease": [
+          "8",
+          "9"
+      ]
+    },
+    { "operatingsystem": "Ubuntu",
+      "operatingsystemrelease": [
+          "12.04",
+          "14.04"
```

Comment:
I don't know, ubuntu part was in borrowed code, probably it's safe to assume it will work